app/components/keyword/keyword.tpl.html

```diff
@@ -1,4 +1,4 @@
 <div class="md-chips">
     <span>{{tag}}</span>
-    <button ng-click="removefn(tag)" ng-if="remove">&times</button>
+    <button ng-click="removefn()" ng-if="remove">&times;</button>
```

Comment:
not quite right. 
read this one to get a better understanding of how this might work:
http://tech.europace.de/passing-functions-to-angularjs-directives/